tool/system_metrics_monitor.py

```diff
@@ -75,7 +76,11 @@ def publish_metrics_to_firestore(metrics):
 
 
 def run_system_metric_monitor(interval=DEFAULT_INTERVAL):
-    initialize_firebase(sys.argv[1])
+    parser = argparse.ArgumentParser(description='Retrieve system metrics i.e cpu utilization, memory & disk usage')
+    parser.add_argument('cert', metavar='cert', type=str, help='path to Firebase JSON certificate keyfile')
```

Comment:
Can you use the same naming scheme we've been using throughout the other scripts? ie. `crypto_token` instead of `cert`/certificate keyfile? e.g. something like in [/firebase_to_json.py](https://github.com/larksystems/nook/blob/master/tool/firebase_to_json.py)

I also don't think you need to set the `metavar` if it's the same as the name?

Reply:
I've changed it to use the same scheme